DigitalLearningSolutions.Data/Services/DelegateApprovalsService.cs

```diff
@@ -183,20 +196,22 @@ private static Email GenerateDelegateApprovalEmail
             var body = new BodyBuilder
             {
                 TextBody = $@"Dear {delegateName},
-                        Your Digital Learning Solutions (DLS) registration at the centre {centreName} has been rejected by an administrator. There are several reasons that this may have happened including:
+                        Your Digital Learning Solutions (DLS) registration at the centre {centreName} has been rejected by an administrator.
+                        There are several reasons that this may have happened including:
                         -You registered with a non-work email address which was not recognised by the administrator
                         -Your DLS centre chooses to manage delegate registration internally
                         -You have accidentally chosen the wrong centre during the registration process.
                         If you need access to the DLS platform, please use the Find Your Centre page to locate your local DLS centre and use the contact details provided there to ask for help with registration. The Find Your Centre page can be found at this URL: {findCentreUrl}",
                 HtmlBody = $@"<body style= 'font - family: Calibri; font - size: small;'>
                                 <p>Dear {delegateName},</p>
-                                <p>Your Digital Learning Solutions (DLS) registration at the centre {centreName} has been rejected by an administrator. There are several reasons that this may have happened including:
+                                <p>Your Digital Learning Solutions (DLS) registration at the centre {centreName} has been rejected by an administrator.</p>
+                                <p>There are several reasons that this may have happened including:
                                     <ul>
                                         <li>You registered with a non-work email address which was not recognised by the administrator</li>
                                         <li>Your DLS centre chooses to manage delegate registration internally</li>
                                         <li>You have accidentally chosen the wrong centre during the registration process.</li>
                                     </ul>
-                                If you need access to the DLS platform, please use the <a href=""{findCentreUrl}"">Find Your Centre page</a> to locate your local DLS centre and use the contact details provided there to ask for help with registration.</p>

```

Comment:
I missed this in the review of the previous version, but \<ul> tags shouldn't be in \<p> tags according to the html spec. Can we close the p above the ul and re-open a new one for the subsequent line, making sure the indentation is updated.

For reference [p](https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element) can only contain [Phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2) elements. [ul](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element) is [Flow content](https://html.spec.whatwg.org/multipage/dom.html#flow-content-2)